Catappult/appcoins-wallet-android
# [APPC-4209] Integrate MiPay

**What does this PR do?**

   Integrate new payment method: mipay

**Database changed?**

   No

**How should this be manually tested?**

  with a gamed injected with xiaomi oemid, try to make a payment in one of the available countries for mipay, like Russia.
  On IAP or onboarding you should have just 1 option to make payments

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-4209](https://aptoide.atlassian.net/browse/APPC-4209)

**Code Review Checklist**



[APPC-4209]: https://aptoide.atlassian.net/browse/APPC-4209?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
